lte/gateway/c/core/oai/tasks/nas/emm/msg/AttachAccept.hpp

```diff
@@ -110,10 +110,16 @@ typedef struct attach_accept_msg_tag {
   additional_update_result_t additionalupdateresult;
 } attach_accept_msg;
 
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
Definition of decode_attach_accept is in https://github.com/magma/magma/pull/13754/files#diff-369802dba7b74897644c3d5932be25dca8c387e8b1248bf27ba5ce3c7d11b8caR35
And prototype is also in hpp file. So do we need to put under extern "C"

Reply:
I am not getting any compilation error/warning for this. It works fine with this change

Reply:
I mean we don't need these changes under  extern C because  implementation and callee of function are in cpp files.